code/modules/mob/living/basic/vermin/mothroach.dm

```diff
@@ -0,0 +1,66 @@
+/mob/living/basic/mothroach
+	name = "mothroach"
+	desc = "An ancient ancestor of the moth that surprisingly looks like the crossbreed of a moth and a cockroach."
+	icon_state = "mothroach"
+	icon_living = "mothroach"
+	icon_dead = "mothroach_dead"
+	held_state = "mothroach"
+	held_lh = 'icons/mob/animal_item_lh.dmi'
+	held_rh = 'icons/mob/animal_item_rh.dmi'
+	head_icon = 'icons/mob/animal_item_head.dmi'
+	butcher_results = list(/obj/item/food/meat/slab/mothroach = 3, /obj/item/stack/sheet/animalhide/mothroach = 1)
+	mob_biotypes = MOB_ORGANIC|MOB_BUG
+	mob_size = MOB_SIZE_SMALL
+	mobility_flags = MOBILITY_FLAGS_REST_CAPABLE_DEFAULT
+	health = 25
+	maxHealth = 25
+	speed = 1.25
+	gold_core_spawnable = FRIENDLY_SPAWN
+	can_be_held = TRUE
+	worn_slot_flags = ITEM_SLOT_HEAD
+
+	verb_say = "flutters"
+	verb_ask = "flutters inquisitively"
+	verb_exclaim = "flutters loudly"
+	verb_yell = "flutters loudly"
+	response_disarm_continuous = "shoos"
+	response_disarm_simple = "shoo"
+	response_harm_continuous = "hits"
+	response_harm_simple = "hit"
+	response_help_continuous = "pats"
+	response_help_simple = "pat"
+	attacked_sound = 'sound/voice/moth/scream_moth.ogg'
+
+	faction = list("neutral")
+
+	ai_controller = /datum/ai_controller/basic_controller/mothroach
+
+/mob/living/basic/mothroach/Initialize(mapload)
+	. = ..()
+	AddElement(/datum/element/pet_bonus, "squeaks happily!")
+	add_verb(src, /mob/living/proc/toggle_resting)
+	ADD_TRAIT(src, TRAIT_VENTCRAWLER_ALWAYS, INNATE_TRAIT)
+
+/mob/living/basic/mothroach/death(gibbed)
+	..()
+	attacked_sound = SFX_PUNCH
```

Comment:
what does this do here

Reply:
Basically, if it's not there, the mothroach will do the moth scream sound when hit, even if dead. 
By adding that, when the mothroach dies, it will swap the screaming for normal punching.

Reply:
they should react to being hit with this sound, they shouldnt replace the hitting sound with it

Reply:
I feel like having the moth scream + punching would be a bit loud or messy. I haven't tested it, but I can give it a try today or tomorrow.

Reply:
I believe the new commit should cover any concerns you've got